tests/helpers.py

```diff
@@ -44,11 +37,11 @@ def run_conda_command(*args):
     # Make sure bin/conda imports *this* conda.
     env['PYTHONPATH'] = os.path.dirname(os.path.dirname(__file__))
     env['CONDARC'] = ' '
-    p= subprocess.Popen((python, conda,) + args, stdout=subprocess.PIPE,
+    p = subprocess.Popen((python, conda,) + args, stdout=subprocess.PIPE,
         stderr=subprocess.PIPE, env=env)
-    stdout, stderr = p.communicate()
-    return (stdout.decode('utf-8').replace('\r\n', '\n'),
-        stderr.decode('utf-8').replace('\r\n', '\n'))
+    stdout, stderr = [stream.strip().decode('utf-8').replace('\r\n', '\n').replace('\\\\', '\\')
```

Comment:
Why should you `replace` double backslashes here? Where do they come from? Which conda command line output produces double backslashes?